docs/podman-info.1.md

```diff
@@ -1,9 +1,9 @@
 % podman-version(1)
 
 ## NAME
+podman\-system\-info - Display system information
```

Comment:
Should this have it's own page, even if it's mostly a cut/paste job?

Reply:
No they are identical,  This is what we do with all of the podman container commands

podman-container-ps -> podman-ps.
for example.  I think we should add the alternate names to the linked images though.